src/os_maild/os_maild_client.c

```diff
@@ -378,77 +379,18 @@ MailMsg *OS_RecvMailQ_JSON(file_queue *fileq, MailConfig *Mail, MailMsg **msg_sm
     if (!(rule = cJSON_GetObjectItem(al_json, "rule"), rule && (mail_flag = cJSON_GetObjectItem(rule, "mail"), mail_flag && cJSON_IsTrue(mail_flag))))
         goto end;
 
+
     /* If e-mail came correctly, generate the e-mail body/subject */
     os_calloc(1, sizeof(MailMsg), mail);
     os_calloc(BODY_SIZE, sizeof(char), mail->body);
     os_calloc(SUBJECT_SIZE, sizeof(char), mail->subject);
 
+    // Add alert to logs
+    char *tab = malloc(14*sizeof(char));
+    strcpy(tab, "\t");
+    cJSON_PrintTable(al_json, logs, body_size, tab, 2);
```

Comment:
Change the name of the function. `cJSON_X()` can lead to a think it belongs to the cJSON library.

Reply:
Done